src/state/connection-mixin.ts

```diff
@@ -156,8 +157,25 @@ export const connectionMixin = <T extends Constructor<HassBaseEl>>(
 
     protected hassReconnected() {
       super.hassReconnected();
+
       this._updateHass({ connected: true });
       broadcastConnectionStatus("connected");
+
+      // @ts-ignore
+      this.hass!.callWS({ type: "get_config" }).then((config: HassConfig) => {
```

Comment:
Don't we already call `get_config` ?

Reply:
oh nm this is `hassReconnected`.

Do we have this logic too for `hassConnected` ?

Reply:
Actually, we use `subscribeConfig` for keeping the main config up to date, so we are already fetching the config there. Now we're fetching it twice ? 

I wonder if we should fire `core_config_updated` event in the backend when the state updates, so we can just listen to `config` updates via the subscription.

Reply:
Yes, that was one of the options I was playing with, and that would also work, but the timing was difficult as you want the state right after the connect.

Reply:
If the state would have updated before connect, you would never get it.

Reply:
The config is automatically refreshed when Home Assistant reconnects. That's built into the collections of Home Assistant Websocket JS: https://github.com/home-assistant/home-assistant-js-websocket/blob/master/lib/collection.ts#L57-L58